driverkit/Makefile

```
@@ -31,7 +31,7 @@ define gen_build_targets
 $(1): $(3)
 	utils/checkfiles $(3)
 	@mkdir -p output/$(2)
-	@${DRIVERKIT} docker -c $(3) --driverversion $(2) --timeout 1000 || echo $(3) >> output/failing.log
+	@${DRIVERKIT} docker -c $(3) --driverversion $(2) --timeout 1000 && { echo $(3) >> output/failing.log; exit 1; }
```

```suggestion
@${DRIVERKIT} docker -c $(3) --driverversion $(2) --timeout 1000 || { echo $(3) >> output/failing.log; exit 1; }
```
so that we exit only when failing build